<ui version="4.0" >
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="labelServer" >
     <property name="text" >
      <string>&amp;Server:</string>
     </property>
     <property name="buddy" >
      <cstring>leServer</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="5" >
    <widget class="QLineEdit" name="leServer" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="labelLogin" >
     <property name="text" >
      <string>&amp;Login:</string>
     </property>
     <property name="buddy" >
      <cstring>leLogin</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QLineEdit" name="leLogin" />
   </item>
   <item row="1" column="4" >
    <widget class="QLabel" name="labelPassword" >
     <property name="text" >
      <string>&amp;Password:</string>
     </property>
     <property name="buddy" >
      <cstring>lePassword</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="5" >
    <widget class="QLineEdit" name="lePassword" >
     <property name="echoMode" >
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label1sPath" >
     <property name="text" >
      <string>&amp;1C path:</string>
     </property>
     <property name="buddy" >
      <cstring>lePath</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4" >
    <widget class="QLineEdit" name="lePath" />
   </item>
   <item row="2" column="5" >
    <widget class="QPushButton" name="pbPath" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="QCheckBox" name="cbMinOnRun" >
     <property name="layoutDirection" >
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text" >
      <string>Minimize on run 1C</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="3" >
    <widget class="QCheckBox" name="cbTrayEnabled" >
     <property name="layoutDirection" >
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text" >
      <string>Use system tray</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="6" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="3" >
    <widget class="QCheckBox" name="cbMinToTray" >
     <property name="layoutDirection" >
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text" >
      <string>Minimize to tray</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3" >
    <widget class="QCheckBox" name="cbStartHidden" >
     <property name="layoutDirection" >
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text" >
      <string>Start hidden</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leServer</tabstop>
  <tabstop>leLogin</tabstop>
  <tabstop>lePassword</tabstop>
  <tabstop>lePath</tabstop>
  <tabstop>pbPath</tabstop>
  <tabstop>cbMinOnRun</tabstop>
  <tabstop>cbTrayEnabled</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
